### DISCUSSION TITLE
N-ary Subtract?

### DISCUSSION THREAD

**Discussion by gramian** [lowdefy/lowdefy#1799] [Questions and Help]
Dear All,
why does the `_subtract` operator only support two arguments, particularly while the `_sum` operator supports many arguments? Couldn't the `_subtract` operator also subtract the 2nd to n-th from the 1st argument array element (similar to Lisp or Scheme)?

Thank you for your insights

**Comment by Gervwyk** [comment]
Hi @gramian 
This would be a nice improvement. And as far as I can see, changing this would not be breaking. We should be able to support this in the next release.
